rpa-home/modules/ROOT/pages/invoke-rpa-from-flow-publish.adoc

```diff
@@ -0,0 +1,26 @@
+
+# Publish an RPA Process to Salesforce
+
+An RPA process is published to Salesforce as an invocable run configuration. Each project can contain one invocable run configuration, which can be used in various other products.
+
+The name of the project, with spaces removed, is contained in the names of the following Salesforce objects:
+
+* Named Credentials 
+* External Credentials
+* External Services
+* MuleSoft Services
+* Invocable Actions
+* Resources
+
+For example, if your project is named `My Super Process`, the corresponding Salesforce object is called `MySuperProcess10'.
+
+The initial values for the Activity Parameters are set in RPA Manager and can be overwritten in Salesforce Flow.
+
+To publish an invocable run configuration to Salesforce, follow these steps in these documentations:
+
+. xref:rpa-manager::processautomation-deploy.adoc#invocable-configuration[Create an Invocable Run Configuration]
```

Comment:
Do you always initial-cap "Invocable Run Configuration"? If not, this should read as if it is text and not a topic title.